src/Tutor.Core/DomainModel/KnowledgeComponents/KnowledgeComponentSatisfied.cs

```diff
@@ -1,10 +1,6 @@
-﻿using Tutor.Core.BuildingBlocks.EventSourcing;
-
-namespace Tutor.Core.DomainModel.KnowledgeComponents
+﻿namespace Tutor.Core.DomainModel.KnowledgeComponents
 {
-    public class KnowledgeComponentSatisfied : DomainEvent
+    public class KnowledgeComponentSatisfied : KnowledgeComponentEvent
```

Comment:
We should move all events into an Events namespace (and folder) inside the KC namespace.

Reply:
I agree, I think both approaches are used for event sourcing in general (having a separate package, and putting them directly inside relevant packages) and some materials actually insist on not putting them in a separate package, but it has cluttered up the KC namespace quite a bit.

Reply:
Events package added in https://github.com/Clean-CaDET/tutor/pull/30/commits/66be496dd6889d39e8527ac4462d8659e3c9d753, this is its current structure: 

![image](https://user-images.githubusercontent.com/57844740/159939286-6d554616-9812-4def-825b-e792ccd19b53.png)

I also moved the AssessmentEventHelpService to the AssessmentEvents package, it seemed like a more appropriate place for it since that's where SubmissionService is.